src/ground_truth_nsrts.py

```diff
@@ -299,11 +303,12 @@ def place_sampler(state: State, rng: np.random.Generator,
 
         def place_on_table_sampler(state: State, rng: np.random.Generator,
                                    objs: Sequence[Object]) -> Array:
-            del state  # unused
+            # Always at the current location.
+            del rng  # this sampler is deterministic
             assert len(objs) == 1
-            lb = 0.0
-            ub = 1.0
-            return np.array(rng.uniform(lb, ub, size=(1, )), dtype=np.float32)
+            held_obj = objs[0]
+            x = state.get(held_obj, "pose") + state.get(held_obj, "grasp")
```

Comment:
just checking that this shouldn't be a minus? Not sure but I always get them mixed up

Reply:
yeah thanks I checked